.github/workflows/e2e-report.yml

```diff
@@ -56,4 +56,4 @@ jobs:
       - name: Deploy e2e page
         if: success()
         run: |
-          ntl deploy --build --cwd e2e-report
+          npx netlify deploy --build --cwd e2e-report
```

Comment:
All the `npx` changes are because when binaries are installed locally, GitHub doesn't automatically add `./node_modules/.bin` to the `PATH`. Just using `npx` is the right approach I think.